run.py

```diff
@@ -32,23 +29,28 @@ def __call__(self, *args, **kwargs):
 
 
 app.config.update(
-    CELERY_BROKER_URL=os.getenv("CELERY_BROKER_URL"),
-    CELERY_RESULT_BACKEND=os.getenv("CELERY_RESULT_BACKEND"),
+    CELERY_BROKER_URL=os.environ["CELERY_BROKER_URL"],
+    CELERY_RESULT_BACKEND=os.environ["CELERY_RESULT_BACKEND"],
```

Comment:
It's safer to use `os.environ.get(...` with default value, and in other places too